specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/IntegrationRuntime.json

```diff
@@ -226,6 +226,74 @@
         {
           "$ref": "#/definitions/IntegrationRuntime"
         }
+      ],
+      "properties": {
+        "typeProperties": {
+          "description": "When this property is not null, means this is a linked integration runtime. The property is used to access original integration runtime.",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/SelfHostedIntegrationRuntimeTypeProperties"
+        }
+      }
+    },
+    "SelfHostedIntegrationRuntimeTypeProperties": {
+      "description": "The self-hosted integration runtime properties.",
+      "type": "object",
+      "properties": {
+        "linkedInfo": {
+          "$ref": "#/definitions/LinkedIntegrationRuntimeProperties"
+        }
+      }
+    },
+    "LinkedIntegrationRuntimeProperties": {
+      "description": "The base definition of a linked integration runtime properties.",
+      "discriminator": "authorizationType",
+      "type": "object",
+      "properties": {
+        "authorizationType": {
+          "type": "string",
+          "description": "The authorization type for integration runtime sharing."
+        }
+      },
+      "required": [
+        "authorizationType"
+      ]
+    },
+    "LinkedIntegrationRuntimeKey": {
+      "x-ms-discriminator-value": "Key",
```

Comment:
>Key [](start = 35, length = 3)

I think this name is not descriptive enough. It would help to see the generated .Net classes. Please  generate the .Net SDK and and a CR.

Reply:
Sure, I will do that